naff/client/client.py

```diff
@@ -193,8 +195,12 @@ def __init__(
         **kwargs,
     ) -> None:
 
-        # Configuration
+        # Set Up logger and overwrite the constant
+        self.logger = logger
+        """The logger NAFF should use"""
+        constants.logger = logger
```

Comment:
How does this interact with multiple clients in the same process?
Can I define a separate logger for each, and will it actually work as expected?

(I assume not, but I feel the need to raise the question)

Reply:
Only one logger is supported, it won't work. Theoretically it should be possible by passing the client obj around to literally every function, but that would be a lot of work to make that change for a very niche feature

Reply:
Pending resolution of this comment before i can merge

Reply:
I'm fine with the answer being "this isn't supported". But a simple mention of that fact in the docs would be appreciated

Reply:
Good point, I will put that in the docstring